wordpress-mobile/WordPress-Android
# Add additional tracking to stats

Fixes #9934 

This PR adds tracking for clicking on the `<` and `>` buttons in the Date selector. We're tracking the stats section where we click - int could be one of the tabs and the Stats detail.

To test:
* Click on the date selector `<` and `>` items
* Check that the events are tracked

Update release notes:
